Source/FieldSolver/WarpXPushFieldsEM.cpp

```diff
@@ -498,7 +526,7 @@ WarpX::ApplyInverseVolumeScalingToCurrentDensity (MultiFab* Jx, MultiFab* Jy, Mu
         // these do not include the guard cells.
         std::array<amrex::Real,3> galilean_shift = {0,0,0};
         const std::array<Real, 3>& xyzmin = WarpX::LowerCorner(tilebox, galilean_shift, lev);
-        const Real rmin  = xyzmin[0] + (tbr.type(0) == NODE ? 0. : 0.5*dx[0]);
+        const Real rmin  = xyzmin[0];
```

Comment:
Could you explain why the above change is needed?

Reply:
This is a bug fix. rmin needs to be the radius at the node, independently of the centering. This was fixed in PR#866

Reply:
This is a bug fix that has been included in PR#866.